### PR TITLE
[MM-9526] Preview text files

### LINKED ISSUE #1265: [Help Wanted] Preview text files
Jira ticket: https://mattermost.atlassian.net/browse/RN-528

We currently support viewing most file types in the app (video, word docs, PDFs, etc), but we still need to add support for previewing text files. 

The expected behaviour should be similar to other files: 
- Tapping on the file thumbnail downloads the file
- The file opens in the previewer so it can be viewed

For text files, the previewer can look similar to the code block preview.

### PR DESCRIPTION
**Pull request by gajananpp**
#### Summary
Allows user to view text file in app

#### Ticket Link
https://mattermost.atlassian.net/browse/RN-528
Closes #1265

#### Checklist
- [x] Has UI changes

#### Device Information
This PR was tested on: [Redmi Note 4, Android 7.0] 

#### Screenshots
Textfile Preview

### CODE REVIEW

File: app/screens/text_preview/text_preview.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    render() {
+        const style = getStyleSheet(this.props.theme);
+
+        return (
+            <ScrollView
+                contentContainerStyle={style.container}
+            >
+                <Text>
```
Comment (enahum) [inline app/screens/text_preview/text_preview.js]: can you add some styling here? perhaps you can use the same one used in the code screen.
https://github.com/mattermost/mattermost-mobile/blob/master/app/screens/code/code.js#L61

**Comment by asaadmahmood** [comment]
@enahum Is there no way to download the file right now?

**Comment by enahum** [comment]
No there is not

**Comment by gajananpp** [comment]
@enahum Added styling to file preview. Used the same as that of code screen
Screenshot

**Comment by gajananpp** [comment]
@asaadmahmood Aligned line numbers on the right side
Screenshot

**Comment by gajananpp** [comment]
@enahum our team uses mattermost to communicate and loves it. Would love to contribute and be a part of it !
Thanks

**Comment by gajananpp** [comment]
@hmhealey done

**Pull request merged**